tests/test_sensor.py

```diff
@@ -160,6 +205,80 @@ def alt_sensors():
     return [Sensor(float, f"name{i}") for i in range(5)]
 
 
+class MyEnum(enum.Enum):
+    ZERO = 0
+    ONE = 1
+
+
+class OtherEnum(enum.Enum):
+    ABC = 0
+    DEF = 1
```

Comment:
Code is easier to follow if you declare things before you use them, even if Python is okay with it.